llvm-external-projects/iree-compiler-api/unittests/tools/compiler_tflite_test.py

```diff
@@ -0,0 +1,105 @@
+# Lint as: python3
+# Copyright 2020 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import logging
+import os
+import sys
+import tempfile
+import unittest
+
+# TODO: No idea why pytype cannot find names from this module.
+# pytype: disable=name-error
+import iree.compiler.tools.tflite
+
+if not iree.compiler.tools.tflite.is_available():
+  print(f"Skipping test {__file__} because the IREE TFLite compiler "
+        f"is not installed")
+  sys.exit(0)
+
+
+class CompilerTest(unittest.TestCase):
+
+  def testImportBinaryPbFile(self):
+    path = os.path.join(os.path.dirname(__file__), "testdata",
+                        "tflite_sample.fb")
+    text = iree.compiler.tools.tflite.compile_file(
+        path, import_only=True).decode("utf-8")
+    logging.info("%s", text)
+    self.assertIn("tosa.mul", text)
+
+  @unittest.skip("IREE tosa compilation not implemented yet")
```

Comment:
still true? (a few in this file)

https://docs.python.org/3/library/unittest.html#skipping-tests-and-expected-failures
could switch to `@unittest.expectedFailure`?